src/LemonWeb/Deployer/Database/SqlUpdate/AbstractSqlUpdate.php

```diff
@@ -29,9 +29,14 @@
      */
     final public function __construct(array $options)
     {
-        $this->options = array_merge($options, array(
-            'charset' => 'utf8',
-        ));
+        $this->options = array_merge(
+            [
+                'charset' => 'utf8',
+                'database_default_name' => 'mijnwebwinkel',
+                'database_lemoncrm_name' => 'administratie',
```

Comment:
Deze database namen horen niet thuis in een publiek toegankelijk pakket, die zullen in de interne code verwerkt moeten worden. Wellicht het beste om deze class te extenden, deze constructor van final af te halen en deze settings in de subclass op te nemen.